### PR TITLE
Update packaging info in contributing.md

### PR DESCRIPTION
**Pull request by micwoj92**
Add basic info on needed tools
Remove github and discord mentions, as they are not needed to package.

### CODE REVIEW

**Review by mastercoms — CHANGES_REQUESTED** [review]
Those should not be removed, given that there should be documentation for future maintainers.

**Comment by micwoj92** [comment]
I only removed them from the contributing.md as they are not needed to package. Future maintainer should look in GitHub, because it is for developers. Removed info is still present in dev/README.md to which I plan rewrite/adding more info about all scripts present there in near future.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -220,20 +220,25 @@ local 32 player bot match on `pl_upward`. (use `+maxplayers 32` in launch option
...
 learn more about the dev scripts in
 [dev/README.md](https://github.com/mastercomfig/mastercomfig/blob/release/dev/README.md).
 
-In order to successfully deploy and announce, you will have to create a new file
-in `dev/` called `mastercomfig-vars`, containing the following variables:
+In order to successfully package presets and addons you need:
+
+* Installed TF2 and Steam with all dependencies
+* bash and basic UNIX tools
+* [VPC](https://developer.valvesoftware.com/wiki/VPK#Linux_.2F_Unix)
```
Comment (mastercoms) [inline .github/CONTRIBUTING.md]: VPK

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (micwoj92) [reply .github/CONTRIBUTING.md]: I copied that from dev/readme.md
I will make that change if you confirm that this needs to be "vpk" there. Thanks for pointing out.

Comment (micwoj92) [reply .github/CONTRIBUTING.md]: I am mostly asking, because in #369 I updated link, but you did not mention it.

Comment (micwoj92) [reply .github/CONTRIBUTING.md]: OH, I just saw it's "VDC" there. My mistake. Should be VDC or VPK then?

Comment (mastercoms) [reply .github/CONTRIBUTING.md]: VPK. There, it was suggesting to see the VDC (Valve Developer Community) for instructions on how to install the vpk tool. Now, it is part of a list of requirements, and thus the actual item being required and thus what should be written is VPK.

Comment (micwoj92) [reply .github/CONTRIBUTING.md]: Done. I may have thought it means like Valve Development sCript? No idea how that came to my mind

File: docs/contributing.md [hunk 2]
```diff
@@ -213,23 +213,40 @@ local 32 player bot match on `pl_upward`. (use `+maxplayers 32` in launch option
...
 ```bash
+#!/bin/sh
+export TF2_DIR="absolute path to your Team Fortress 2 directory"
+```
+## Release and announce scripts
+
+If you want to test the scripts that upload to github and/or announce in discord, 
+you should append following to your `mastercomfig-vars` file:
+
+```
```
[the remark this answers is not in the document]
Comment (micwoj92) [reply docs/contributing.md]: bro, this is draft, hold on

**Comment by micwoj92** [comment]
After thinking a bit about this  I came to conclusion that some might want to contribute to the release/announce scripts, so I added them in separate paragraph. Is this good @mastercoms ?

**Review by mastercoms — APPROVED** [review]

**Pull request merged**